src/test/java/test/SynchronyTest.java

```diff
@@ -38,4 +38,21 @@ void synchrony_enable(Product product) throws Exception {
                 .hasTextContaining("-Dsynchrony.service.url=https://mysynchrony/v1")
                 .hasTextNotContaining("synchrony.btf.disabled");
     }
+
+    @ParameterizedTest
```

Comment:
Thanks for creating the test. ⭐ 

Minor - we don't really need it parametrized as it will have only one execution for Confluence. But I can see we use `@ParameterizedTest` above as well so we can keep it for consistency.